### PR TITLE
refactor: improve sanskrit translation

### LINKED ISSUE #1524: 🐛 Bug Report: Issues in Sanskrit Translation of notification-center
### 📜 Description

1. Translation for `notifications` is wrong
```js
notifications: "विज्ञप्तयः",
```
directly translates to "Announcements"

2. Translation for `poweredBy` is wrong
```js
    poweredBy: 'नियन्त्रणकृत्'
```
directly translates to "Controller"

### 👟 Reproduction steps

None

### 👍 Expected behavior

The translation should be accurate.

### 👎 Actual Behavior with Screenshots

The translation is quite misleading.

### 💻 Operating system

Linux

### 🤖 Node Version

18.10.0

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by thenishantsapkota**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Improves Sanskrit Translation

- **Why was this change needed?** (You can also link to an open issue here)
- For internationalization

- Resolves #1524

### CODE REVIEW

**Comment by thenishantsapkota** [comment]
@p-fernandez  Please review the changes

File: packages/notification-center/src/i18n/languages/sa.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { ITranslationEntry } from '../lang';
 
 export const SA: ITranslationEntry = {
   translations: {
-    notifications: 'विज्ञप्तयः',
+    notifications: 'अधिसूचनाः',
     markAllAsRead: 'सर्वं पठितम् इति सूचनीयम्',
-    poweredBy: 'नियन्त्रणकृत्',
+    poweredBy: 'द्वारा संचालितः',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/sa.ts:7]: Are the colons needed? I do not know sanskrit so I don't if it is part of the a sanskrit character or just adding the colons as a punctuation mark.

Comment (thenishantsapkota) [reply packages/notification-center/src/i18n/languages/sa.ts:7]: It is the part of the sanskrit language so it needs to be there!

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**